etc/config/c++.amazon.properties

```diff
@@ -991,11 +991,11 @@ libs.ctre.versions.v2.version=v2
 libs.ctre.versions.v2.path=/opt/compiler-explorer/libs/ctre/v2/include
 libs.spy.name=SPY
 libs.spy.description=C++17 constexpr settings detectors
-libs.spy.versions=trunk:v003
+libs.spy.versions=trunk:v004
 libs.spy.url=https://github.com/jfalcou/spy
 libs.spy.versions.trunk.version=trunk
 libs.spy.versions.trunk.path=/opt/compiler-explorer/libs/spy/trunk/include
-libs.spy.versions.v003.version=v0.0.3
+libs.spy.versions.v003.version=v0.0.4
```

Comment:
Needs to be v004 instead of v003. Also, can you add libs.spy.versions.v004.alias=v003, so that existing links that include the old version still work

Reply:
Oh ! didn't thought about links. Let's fix this